stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -498,6 +498,17 @@
         <item name="streamUiMediaAttachmentMoreCountTextStyle">normal</item>
     </style>
 
+    <style name="StreamUi.MessageList.GiphyMediaAttachment" parent="StreamUi">
+        <item name="streamUiGiphyMediaAttachmentProgressIcon">@drawable/stream_ui_rotating_indeterminate_progress_gradient</item>
+        <item name="streamUiGiphyMediaAttachmentGiphyIcon">@drawable/stream_ui_giphy_label</item>
+        <item name="streamUiGiphyMediaAttachmentPlaceHolderIcon">@drawable/stream_ui_picture_placeholder</item>
+        <item name="streamUiGiphyMediaAttachmentImageBackgroundColor">@color/stream_ui_literal_transparent</item>
+        <item name="streamUiGiphyMediaAttachmentConstantSize">false</item>
+        <item name="streamUiGiphyMediaAttachmentGiphyMaxHeight">200dp</item>
```

Comment:
We could consider providing users with a way of setting width as well. It's easy to implement however the issue is that this would become irrelevan when `streamUiGiphyMediaAttachmentConstantSize` is set to `false` so it might mislead.

Currently what happens is that if `streamUiGiphyMediaAttachmentConstantSize` is set to `true` the Giphy container is a square where `heighet == width ==  streamUiGiphyMediaAttachmentGiphyMaxHeight.`

Reply:
width is set as the maximum width allowed for messages, in the parent. It's kind of hard to do both and expect working results in all scenarios